src/components/uploader/js/FileQueueView.js

```diff
@@ -394,7 +394,6 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
 
         strings: {
             progress: {
-                toUploadLabel: "To upload: %fileCount %fileLabel (%totalBytes)",
```

Comment:
This string was removed but the use of it wasn't. I think a better role for this pull request would be to add tests for the functionality that it has rather than removing the functionality